### ISSUE TITLE
hover doen not work in jupyter notebook when use pylance server

### ISSUE THREAD

**Issue by CatNofishing** [microsoft/pylance-release#756]
## Environment data
version: 1.52.1 (user setup)
commit: ea3859d4ba2f3e577a159bc91e3074c5d85c0523
date: 2020-12-16T16:34:46.910Z
Electron: 9.3.5
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.18363

##issue screen

![image](https://user-images.githubusercontent.com/33288114/102730022-11430300-436e-11eb-978b-ceefa7a585d6.pn

**Comment by jakebailey** [comment]
Is this the new notebooks view, or the old one? The updated one should work, but I don't believe the older webview-based one works with hover.

**Comment by CatNofishing** [comment]
Yes, it is the old one jupyter in latest stable version vscode,

**Comment by jakebailey** [comment]
Yeah, I don't think the old one will work. Can you try out the new notebook view, which shouldn't have that limitation?

**Comment by CatNofishing** [comment]
Do you mean the new jupyter is in insider vscode? There are still many problems with that version of jupyter.

**Comment by jakebailey** [comment]
Yes, but it's the more featureful of the two given it plugs directly into VS Code's UI, hence why I am asking.

**Comment by CatNofishing** [comment]
Hi, any plane to solve this issue?

**Comment by jakebailey** [comment]
I don't think there is anything we can do on this tracker; we will answer any hover call we get, so it is up to the Jupyter extension to call the code. My impression is that they get this feature "for free" with new notebooks, versus old notebooks missing this due to it needing to be implemented from scratch.

I'm going to transfer this issue, since we can't affect the UI that calls us (and we can figure out what's going on with the right people present).
